plugins/youtube/plugin.py

```diff
@@ -88,7 +88,7 @@ def _get_video_info(self, cardinal, channel, url):
             message = self._parse_item(result['items'][0])
             cardinal.sendMsg(channel, message)
         except IndexError:
-            raise EventRejectedMessage
+            raise EventRejectedMessage, e
```

Comment:
I don't think "e" needs to be raised here. For one thing "e" doesn't get created because except doesn't save it as "e", but this is an expected exception, and [EventRejectedMessage is a special Cardinal exception](https://github.com/JohnMaguire/Cardinal/wiki/Events#refusing-events) indicating a rejection of the event so that other plugins may process it.

If you agree, I can remove this myself before merging. :)